### ISSUE TITLE
There is browser leaking problem: a lot of orphaned browser instances

### ISSUE THREAD

**Issue by pftg** [mattheworiordan/capybara-screenshot#271]
On each new screenshots there are new 7 chrome instances. So for 10 tests we will have 70 chrome instances.

For now I have not debug the root of the problem. Will left it here.

This is happens for Firefox too, but Chrome ate more memory, so on CI there are not much resources to host so many instances.

**Comment by jason-rutherford** [comment]
There's not much information to go off of here but after stumbling upon this issue I thought I'd share something in case it helps others. 

We have been seeing a lot of orphaned chrome browsers. After seeing this issue we realized that we do not use the screenshots generated from that Jenkins environment so we just disabled them on that environment.

Then I noticed that often flaky failing specs would open a new window for a "printable view", without toolbars, and that new browser window renders a report-like page with javascript that automatically opens the print dialog. So I'm not sure if capybara-screenshot is related to the orphaned chrome browsers, especially when I found this release note item in a newer [ChromeDriver](https://chromedriver.chromium.org/downloads) 84.0.4147.30:

- Resolved issue 3420: after switching to the print window, the chromedriver stops responding

We will either update chrome and chromedriver or rethink that automatic print dialog.
